spring-graphql/src/test/java/org/springframework/graphql/data/method/annotation/support/ArgumentMethodArgumentResolverTests.java

```diff
@@ -179,6 +258,14 @@ public Long getAuthorId() {
 		public void setAuthorId(Long authorId) {
 			this.authorId = authorId;
 		}
+
+		public OptionalInput<String> getNotes() {
+			return this.notes;
+		}
+
+		public void setNotes(OptionalInput<String> notes) {
+			this.notes = (notes == null) ? OptionalInput.defined(null) : notes;
```

Comment:
`DataBinder.bind` doesn't pass `null` values to the conversion service, so this setter can receive `null`.
The condition here is more like a quickfix, and I hope someone knows a better way around this.